config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```

Comment:
If it's part of `ember init`, I'd vote for leaving it in.  We don't explain everything about Ember during the tutorial and this is probably something to skip ...